lms/authentication/password_hash.py

```diff
@@ -34,7 +34,7 @@ def hash_password(password: str, salt: str = ""):
     if isinstance(password, str):
         password = password.encode("utf8")
 
-    if salt == "":
```

Comment:
PyLint wants you to use the falsiness of False, None, empty string, empty list, etc, instead of comparing to them